### ISSUE TITLE
bash: /usr/libexec/vte-urlencode-cwd: No such file or directory

### ISSUE THREAD

**Issue by juhp** [containers/toolbox#390]
When I tried to start a new fedora-toolbox:32 with toolbox-0.0.18-2.fc32:

```
[petersen@localhost ~]$ toolbox create --candidate-registry 
Image required to create toolbox container.
Download candidate-registry.fedoraproject.org/f32/fedora-toolbox:32 (500MB)? [y/N]: y
Created container: fedora-toolbox-32
Enter with: toolbox enter
[petersen@localhost ~]$ toolbox enter

Welcome to the Toolbox; a container where you can install and run
all your tools.

 - Use DNF in the usual manner to install command line tools.
 - To create a new tools container, run 'toolbox create'.

For more information, see the documentation.

bash: /usr/libexec/vte-urlencode-cwd: No such file or directory
⬢[petersen@toolbox ~]$ 
```

I see that `vte291` provides that file, but I don't know the full story behind this warning.

**Comment by HarryMichal** [comment]
I saw something similar mentioned in Fedora Discord channel. But if it's related to VTE then that's a question for @debarshiray.

**Comment by ondrejholy** [comment]
Just note that `dnf install vte291` inside the container fixes the issue.

**Comment by juhp** [comment]
Yeah, before adding vte291, it would be good to understand what changed

**Comment by juhp** [comment]
Yes, I think `/usr/libexec/vte-urlencode-cwd` should be moved from `vte291` to the `vte-profile` subpackage.  So I filed <https://bugzilla.redhat.com/show_bug.cgi?id=1815769>.

**Comment by junjieyuan** [comment]
fixed after dnf upgrade vte-profile

```text
⬢[junjie@toolbox ~]$ sudo dnf upgrade
sudo: setrlimit(RLIMIT_CORE): Operation not permitted
Fedora Modular 32 - x86_64                                                         908  B/s | 5.0 kB     00:05    
Fedora Modular 32 - x86_64                                                          20 kB/s | 188 kB     00:09    
Fedora Modular 32 - x86_64 - Updates                                               1.5 kB/s | 5.5 kB     00:03    
Fedora Modular 32 - x86_64 - Test Updates                                          544  B/s | 4.5 kB     00:08    
Fedora Modular 32 - x86_64 - Test Updates                                           22 kB/s | 172 kB     00:07    
Fedora 32 - x86_64 - Test Updates                                                  2.4 kB/s | 5.1 kB     00:02    
Fedora 32 - x86_64 - Test Updates                                                   21 kB/s | 1.9 MB     01:30    
Fedora 32 - x86_64 - Updates                                                       3.0 kB/s | 5.5 kB     00:01    
Fedora 32 - x86_64                                                                 5.5 kB/s |  23 kB     00:04    
Fedora 32 - x86_64                                                                 135 kB/s | 2.7 MB     00:20    
Dependencies resolved.
===================================================================================================================
 Package                            Architecture     Version                       Repository                 Size
===================================================================================================================
Upgrading:
 coreutils                          x86_64           8.32-3.fc32.1                 updates-testing           1.2 M
 coreutils-common                   x86_64           8.32-3.fc32.1                 updates-testing           2.0 M
 cyrus-sasl-lib                     x86_64           2.1.27-4.fc32                 updates-testing           100 k
 fedora-release-common              noarch           32-0.8                        updates-testing            20 k
 fedora-release-container           noarch           32-0.8                        updates-testing            13 k
 git                                x86_64           2.26.0-0.3.rc2.fc32           updates-testing           127 k
 git-core                           x86_64           2.26.0-0.3.rc2.fc32           updates-testing           4.8 M
 git-core-doc                       noarch           2.26.0-0.3.rc2.fc32           updates-testing           2.2 M
 libblkid                           x86_64           2.35.1-6.fc32                 updates-testing           153 k
 libfdisk                           x86_64           2.35.1-6.fc32                 updates-testing           204 k
 libmount                           x86_64           2.35.1-6.fc32                 updates-testing           180 k
 libreport-filesystem               noarch           2.12.0-3.fc32                 updates-testing            12 k
 libsmartcols                       x86_64           2.35.1-6.fc32                 updates-testing           121 k
 libuuid                            x86_64           2.35.1-6.fc32                 updates-testing            28 k
 perl-Git                           noarch           2.26.0-0.3.rc2.fc32           updates-testing            46 k
 systemd                            x86_64           245.2-1.fc32                  updates-testing           4.2 M
 systemd-libs                       x86_64           245.2-1.fc32                  updates-testing           558 k
 systemd-pam                        x86_64           245.2-1.fc32                  updates-testing           292 k
 systemd-rpm-macros                 noarch           245.2-1.fc32                  updates-testing            25 k
 util-linux                         x86_64           2.35.1-6.fc32                 updates-testing           2.7 M
 vte-profile                        x86_64           0.60.0-2.fc32                 updates-testing            16 k

Transaction Summary
===================================================================================================================
Upgrade  21 Packages

Total download size: 19 M
Is this ok [y/N]: y
Downloading Packages:
(1/21): cyrus-sasl-lib-2.1.27-4.fc32.x86_64.rpm                                    4.8 kB/s | 100 kB     00:20    
(2/21): fedora-release-common-32-0.8.noarch.rpm                                    2.7 kB/s |  20 kB     00:07    
(3/21): fedora-release-container-32-0.8.noarch.rpm                                  11 kB/s |  13 kB     00:01    
(4/21): git-2.26.0-0.3.rc2.fc32.x86_64.rpm                                          12 kB/s | 127 kB     00:10    
(5/21): coreutils-common-8.32-3.fc32.1.x86_64.rpm                                   25 kB/s | 2.0 MB     01:21    
(6/21): coreutils-8.32-3.fc32.1.x86_64.rpm                                          13 kB/s | 1.2 MB     01:32    
(7/21): libblkid-2.35.1-6.fc32.x86_64.rpm                                           25 kB/s | 153 kB     00:06    
(8/21): libfdisk-2.35.1-6.fc32.x86_64.rpm                                           18 kB/s | 204 kB     00:11    
(9/21): libmount-2.35.1-6.fc32.x86_64.rpm                                           25 kB/s | 180 kB     00:07    
(10/21): libreport-filesystem-2.12.0-3.fc32.noarch.rpm                              16 kB/s |  12 kB     00:00    
(11/21): libsmartcols-2.35.1-6.fc32.x86_64.rpm                                      25 kB/s | 121 kB     00:04    
(12/21): libuuid-2.35.1-6.fc32.x86_64.rpm                                          8.5 kB/s |  28 kB     00:03    
(13/21): perl-Git-2.26.0-0.3.rc2.fc32.noarch.rpm                                   6.4 kB/s |  46 kB     00:07    
(14/21): git-core-doc-2.26.0-0.3.rc2.fc32.noarch.rpm                                27 kB/s | 2.2 MB     01:20    
(15/21): systemd-libs-245.2-1.fc32.x86_64.rpm                                       22 kB/s | 558 kB     00:24    
(16/21): systemd-pam-245.2-1.fc32.x86_64.rpm                                        12 kB/s | 292 kB     00:24    
(17/21): systemd-rpm-macros-245.2-1.fc32.noarch.rpm                                8.9 kB/s |  25 kB     00:02    
(18/21): util-linux-2.35.1-6.fc32.x86_64.rpm                                        22 kB/s | 2.7 MB     02:04    
(19/21): vte-profile-0.60.0-2.fc32.x86_64.rpm                                       23 kB/s |  16 kB     00:00    
(20/21): systemd-245.2-1.fc32.x86_64.rpm                                            14 kB/s | 4.2 MB     05:02    
(21/21): git-core-2.26.0-0.3.rc2.fc32.x86_64.rpm                                    11 kB/s | 4.8 MB     07:45    
-------------------------------------------------------------------------------------------------------------------
Total                                                                               38 kB/s |  19 MB     08:30     
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                           1/1 
  Running scriptlet: libuuid-2.35.1-6.fc32.x86_64                                                              1/1 
  Upgrading        : libuuid-2.35.1-6.fc32.x86_64                                                             1/42 
  Upgrading        : git-core-2.26.0-0.3.rc2.fc32.x86_64                                                      2/42 
  Upgrading        : git-core-doc-2.26.0-0.3.rc2.fc32.noarch                                                  3/42 
  Upgrading        : perl-Git-2.26.0-0.3.rc2.fc32.noarch                                                      4/42 
  Upgrading        : git-2.26.0-0.3.rc2.fc32.x86_64                                                           5/42 
  Upgrading        : fedora-release-container-32-0.8.noarch                                                   6/42 
  Upgrading        : fedora-release-common-32-0.8.noarch                                                      7/42 
  Upgrading        : systemd-rpm-macros-245.2-1.fc32.noarch                                                   8/42 
  Upgrading        : libsmartcols-2.35.1-6.fc32.x86_64                                                        9/42 
  Upgrading        : coreutils-common-8.32-3.fc32.1.x86_64                                                   10/42 
  Upgrading        : coreutils-8.32-3.fc32.1.x86_64                                                          11/42 
  Upgrading        : libblkid-2.35.1-6.fc32.x86_64                                                           12/42 
  Running scriptlet: libblkid-2.35.1-6.fc32.x86_64                                                           12/42 
  Upgrading        : libfdisk-2.35.1-6.fc32.x86_64                                                           13/42 
  Upgrading        : libmount-2.35.1-6.fc32.x86_64                                                           14/42 
  Upgrading        : systemd-libs-245.2-1.fc32.x86_64                                                        15/42 
  Running scriptlet: systemd-libs-245.2-1.fc32.x86_64                                                        15/42 
  Upgrading        : util-linux-2.35.1-6.fc32.x86_64                                                         16/42 
  Running scriptlet: util-linux-2.35.1-6.fc32.x86_64                                                         16/42 
  Upgrading        : systemd-pam-245.2-1.fc32.x86_64                                                         17/42 
  Running scriptlet: systemd-245.2-1.fc32.x86_64                                                             18/42 
  Upgrading        : systemd-245.2-1.fc32.x86_64                                                             18/42 
  Running scriptlet: systemd-245.2-1.fc32.x86_64                                                             18/42 
  Upgrading        : vte-profile-0.60.0-2.fc32.x86_64                                                        19/42 
  Upgrading        : libreport-filesystem-2.12.0-3.fc32.noarch                                               20/42 
  Upgrading        : cyrus-sasl-lib-2.1.27-4.fc32.x86_64                                                     21/42 
  Cleanup          : git-2.25.1-3.fc32.x86_64                                                                22/42 
  Cleanup          : fedora-release-common-32-0.6.noarch                                                     23/42 
  Cleanup          : git-core-doc-2.25.1-3.fc32.noarch                                                       24/42 
  Cleanup          : fedora-release-container-32-0.6.noarch                                                  25/42 
  Cleanup          : perl-Git-2.25.1-3.fc32.noarch                                                           26/42 
  Cleanup          : vte-profile-0.60.0-1.fc32.x86_64                                                        27/42 
  Cleanup          : libreport-filesystem-2.12.0-2.fc32.noarch                                               28/42 
  Running scriptlet: systemd-245~rc1-4.fc32.x86_64                                                           29/42 
  Cleanup          : systemd-245~rc1-4.fc32.x86_64                                                           29/42 
  Running scriptlet: systemd-245~rc1-4.fc32.x86_64                                                           29/42 
  Cleanup          : util-linux-2.35.1-5.fc32.x86_64                                                         30/42 
  Cleanup          : systemd-rpm-macros-245~rc1-4.fc32.noarch                                                31/42 
  Cleanup          : libfdisk-2.35.1-5.fc32.x86_64                                                           32/42 
  Cleanup          : libmount-2.35.1-5.fc32.x86_64                                                           33/42 
  Cleanup          : libblkid-2.35.1-5.fc32.x86_64                                                           34/42 
  Cleanup          : systemd-libs-245~rc1-4.fc32.x86_64                                                      35/42 
  Cleanup          : coreutils-8.32-3.fc32.x86_64                                                            36/42 
  Cleanup          : coreutils-common-8.32-3.fc32.x86_64                                                     37/42 
  Cleanup          : libuuid-2.35.1-5.fc32.x86_64                                                            38/42 
  Cleanup          : libsmartcols-2.35.1-5.fc32.x86_64                                                       39/42 
  Cleanup          : systemd-pam-245~rc1-4.fc32.x86_64                                                       40/42 
  Cleanup          : git-core-2.25.1-3.fc32.x86_64                                                           41/42 
  Cleanup          : cyrus-sasl-lib-2.1.27-3.fc32.x86_64                                                     42/42 
  Running scriptlet: cyrus-sasl-lib-2.1.27-3.fc32.x86_64                                                     42/42 
  Verifying        : coreutils-8.32-3.fc32.1.x86_64                                                           1/42 
  Verifying        : coreutils-8.32-3.fc32.x86_64                                                             2/42 
  Verifying        : coreutils-common-8.32-3.fc32.1.x86_64                                                    3/42 
  Verifying        : coreutils-common-8.32-3.fc32.x86_64                                                      4/42 
  Verifying        : cyrus-sasl-lib-2.1.27-4.fc32.x86_64                                                      5/42 
  Verifying        : cyrus-sasl-lib-2.1.27-3.fc32.x86_64                                                      6/42 
  Verifying        : fedora-release-common-32-0.8.noarch                                                      7/42 
  Verifying        : fedora-release-common-32-0.6.noarch                                                      8/42 
  Verifying        : fedora-release-container-32-0.8.noarch                                                   9/42 
  Verifying        : fedora-release-container-32-0.6.noarch                                                  10/42 
  Verifying        : git-2.26.0-0.3.rc2.fc32.x86_64                                                          11/42 
  Verifying        : git-2.25.1-3.fc32.x86_64                                                                12/42 
  Verifying        : git-core-2.26.0-0.3.rc2.fc32.x86_64                                                     13/42 
  Verifying        : git-core-2.25.1-3.fc32.x86_64                                                           14/42 
  Verifying        : git-core-doc-2.26.0-0.3.rc2.fc32.noarch                                                 15/42 
  Verifying        : git-core-doc-2.25.1-3.fc32.noarch                                                       16/42 
  Verifying        : libblkid-2.35.1-6.fc32.x86_64                                                           17/42 
  Verifying        : libblkid-2.35.1-5.fc32.x86_64                                                           18/42 
  Verifying        : libfdisk-2.35.1-6.fc32.x86_64                                                           19/42 
  Verifying        : libfdisk-2.35.1-5.fc32.x86_64                                                           20/42 
  Verifying        : libmount-2.35.1-6.fc32.x86_64                                                           21/42 
  Verifying        : libmount-2.35.1-5.fc32.x86_64                                                           22/42 
  Verifying        : libreport-filesystem-2.12.0-3.fc32.noarch                                               23/42 
  Verifying        : libreport-filesystem-2.12.0-2.fc32.noarch                                               24/42 
  Verifying        : libsmartcols-2.35.1-6.fc32.x86_64                                                       25/42 
  Verifying        : libsmartcols-2.35.1-5.fc32.x86_64                                                       26/42 
  Verifying        : libuuid-2.35.1-6.fc32.x86_64                                                            27/42 
  Verifying        : libuuid-2.35.1-5.fc32.x86_64                                                            28/42 
  Verifying        : perl-Git-2.26.0-0.3.rc2.fc32.noarch                                                     29/42 
  Verifying        : perl-Git-2.25.1-3.fc32.noarch                                                           30/42 
  Verifying        : systemd-245.2-1.fc32.x86_64                                                             31/42 
  Verifying        : systemd-245~rc1-4.fc32.x86_64                                                           32/42 
  Verifying        : systemd-libs-245.2-1.fc32.x86_64                                                        33/42 
  Verifying        : systemd-libs-245~rc1-4.fc32.x86_64                                                      34/42 
  Verifying        : systemd-pam-245.2-1.fc32.x86_64                                                         35/42 
  Verifying        : systemd-pam-245~rc1-4.fc32.x86_64                                                       36/42 
  Verifying        : systemd-rpm-macros-245.2-1.fc32.noarch                                                  37/42 
  Verifying        : systemd-rpm-macros-245~rc1-4.fc32.noarch                                                38/42 
  Verifying        : util-linux-2.35.1-6.fc32.x86_64                                                         39/42 
  Verifying        : util-linux-2.35.1-5.fc32.x86_64                                                         40/42 
  Verifying        : vte-profile-0.60.0-2.fc32.x86_64                                                        41/42 
  Verifying        : vte-profile-0.60.0-1.fc32.x86_64                                                        42/42 

Upgraded:
  coreutils-8.32-3.fc32.1.x86_64                         coreutils-common-8.32-3.fc32.1.x86_64                    
  cyrus-sasl-lib-2.1.27-4.fc32.x86_64                    fedora-release-common-32-0.8.noarch                      
  fedora-release-container-32-0.8.noarch                 git-2.26.0-0.3.rc2.fc32.x86_64                           
  git-core-2.26.0-0.3.rc2.fc32.x86_64                    git-core-doc-2.26.0-0.3.rc2.fc32.noarch                  
  libblkid-2.35.1-6.fc32.x86_64                          libfdisk-2.35.1-6.fc32.x86_64                            
  libmount-2.35.1-6.fc32.x86_64                          libreport-filesystem-2.12.0-3.fc32.noarch                
  libsmartcols-2.35.1-6.fc32.x86_64                      libuuid-2.35.1-6.fc32.x86_64                             
  perl-Git-2.26.0-0.3.rc2.fc32.noarch                    systemd-245.2-1.fc32.x86_64                              
  systemd-libs-245.2-1.fc32.x86_64                       systemd-pam-245.2-1.fc32.x86_64                          
  systemd-rpm-macros-245.2-1.fc32.noarch                 util-linux-2.35.1-6.fc32.x86_64                          
  vte-profile-0.60.0-2.fc32.x86_64                      

Complete!
```

**Comment by juhp** [comment]
Correct, fixed in `vte-profile-0.60.0-2.fc32`.
